java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -141,7 +141,11 @@ class DatabaseImpl(
         }
     }
 
-    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) = Unit
+    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) = db.transaction {
+        ((oldVersion + 1)..newVersion).forEach {
+            nextVersion -> MIGRATION_STEPS[nextVersion]?.forEach(db::execSQL)
+        }
```

Comment:
Lambda arg goes on the same line as the opening brace.
```suggestion
        ((oldVersion + 1)..newVersion).forEach { nextVersion ->
            MIGRATION_STEPS[nextVersion]?.forEach(db::execSQL)
        }
```